qutebrowser/browser/webelem.py

```diff
@@ -450,6 +450,10 @@ def hover(self) -> None:
                             Qt.NoModifier)
         self._tab.send_event(event)
 
+    def info_str(self) -> Union[str, None]:
```

Comment:
```suggestion
    def info_str(self) -> Optional[str]:
```

Reply:
Hi!

id is a good addition too.

I've also been thinking about it, and I agree it would make sense to separate it out from hover because then hover and info could hint different elements. It makes sense for hover to only hint the elements it currently hints.

For info hinting, I'd also like to get title/alt info about non clickable images and any other element that has them. I have to use an internal tool where I cannot remember what any of the abbreviations/columns/buttons/icons/etc mean but there's an entire cheatsheet hidden away in the title tags on these elements.

I'll make the change you suggested and then see if I can do the rest when I have time.